### PR TITLE
Duckpan New promoted for Spice

### PR DESCRIPTION
**Pull request by talsraviv**

### CODE REVIEW

**Comment by talsraviv** [comment]
@moollaza let me know what you think of this, and I'll do this for goodies as well

**Comment by moollaza** [comment]
@talsraviv LGTM :+1: 

Only thought is that the title shouldn't be about `duckpan new` -- it still deserves a heading, but I think the main heading for the page should speak to the fact that you're looking at a Spice tutorial?

**Comment by talsraviv** [comment]
@moollaza added to Goodies

**Comment by moollaza** [comment]
LGTM :-1:

File: duckduckhack/spice/spice_basic_tutorial.md [hunk 1]
```diff
@@ -1,12 +1,46 @@
...
+2. At the command line, type `duckpan new` and hit enter.
+3. When prompted, enter what you want to call your Spice Instant Answer.
+
+	*For example, `ron swanson quotes`. White spaces and character casing are automatically formatted.*
+
+4. You will see a list of the boilerplate files created for you, each with the proper naming conventions already done:
 
-# NPM Spice - Backend (Perl)
+	```
+	[09:22 PM ... zeroclickinfo-spice {master}]$ duckpan new
```
Comment (moollaza) [inline duckduckhack/spice/spice_basic_tutorial.md]: Just noticed there's lots of trailing spaces here. Please remove them. It's not a huge problem, just that it extends the scrollbar on the codeblock so it looks funny.

**Comment by talsraviv** [comment]
Did right on master

**Pull request merged**